modules/react/common/lib/theming/useTheme.ts

```diff
@@ -45,7 +44,10 @@ export function useTheme(theme?: PartialEmotionCanvasTheme): EmotionCanvasTheme
     // Context not supported or invalid (probably called from within a class component)
   }
 
-  const windowTheme = typeof window !== 'undefined' && get(window, 'window.workday.canvas.theme');
+  // This CK support chaining?
```

Comment:
Question for the devs to avoid the whole checking everything

Reply:
yeah this i probably the only place in our code that has some workday specific code,